Sources/JSON.swift

```diff
@@ -0,0 +1,50 @@
+public enum JSON {
+    public enum Number {
+        case integer(Int)
+        case unsignedInteger(UInt)
```

Comment:
It's possible. If the number starts with `-` it is stored as signed. If not it is stored as unsigned.